src/utils/_variables.scss

```diff
@@ -0,0 +1,9 @@
+$block-height: 300px;
+
+$colors:
+  "1" rgb(255, 0, 0),
+  "2" rgb(204, 0, 0),
+  "3" rgb(153, 0, 0),
+  "4" rgb(102, 0, 0),
+  "5" rgb(51, 0, 0),
+  "6" rgb(0, 0, 0);
```

Comment:
Let's try to generate this colors using scss loops :D

Reply:
> You can try to generate these colors using scss loops :D

nice suggestion🔥🔥🔥